CHANGELOG.md

```diff
@@ -3,5 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.2.18.1.0
+- Fix bidding ads.
```

Comment:
Should this be something like "Add support for bidding ads" instead?

Reply:
Yeah. When I wrote that, I was thinking of how support for bidding ads had been implicit because the Android adapter supported them but that was never specified so it's not actually like support was "broken", it was just absent.